Formula/tart.rb

```diff
@@ -0,0 +1,37 @@
+class Tart < Formula
+  desc "macOS and Linux VMs on Apple Silicon to use in CI and other automations"
+  homepage "https://github.com/cirruslabs/tart#readme"
+  url "https://github.com/cirruslabs/tart/archive/refs/tags/0.33.0.tar.gz"
+  sha256 "66c2b3a0b175344bdb2f9a61c7c34e9be935a7f241b3fd44721fe995852581bd"
+  license "AGPL-3.0-or-later"
+
+  depends_on "rust" => :build
+  depends_on xcode: ["14.1", :build]
+  depends_on arch: :arm64
+  depends_on macos: :monterey
+  depends_on :macos
+
+  uses_from_macos "swift"
+
+  resource "softnet" do
+    url "https://github.com/cirruslabs/softnet/archive/refs/tags/0.3.0.tar.gz"
+    sha256 "b77fb9424cf6a5c61e2a2513a531f0a9321047890979ca7282a5c036ebf48938"
+  end
+
+  def install
+    resource("softnet").stage do
+      system "cargo", "install", *std_cargo_args
+    end
+    system "swift", "build", "--disable-sandbox", "-c", "release"
+    system "/usr/bin/codesign", "-f", "-s", "-", "--entitlement", "Resources/tart.entitlements", ".build/release/tart"
```

Comment:
This should really be fixed upstream instead of having Homebrew sign files

Reply:
If adding the following Makefile to upstream, will this formula be accepted?

```makefile
PREFIX ?= /usr/local

.PHONY: all build install

all: build

build:
	swift build --disable-sandbox -c release
	codesign -f -s - --entitlement Resources/tart.entitlements .build/release/tart

install: all
	mkdir -p "$(PREFIX)/bin"
	cp -f .build/release/tart "$(PREFIX)/bin/tart"
```